src/screens/users/list/Users.tsx

```diff
@@ -125,14 +139,17 @@ export default class Users extends BaseAutoRefreshScreen<Props, State> {
     const style = computeStyleSheet();
     const { users, count, skip, limit, refreshing, loading } = this.state;
     const { navigation } = this.props;
+    const carFullName = this.carFullName;
+    const carLicensePlate = this.carLicensePlate;
     return (
       <Container style={style.container}>
         <HeaderComponent
-          title={i18n.t('sidebar.users')}
+          title={carFullName ?? i18n.t('sidebar.users')}
+          subTitle={carLicensePlate}
+          subTitleSecondLine={count > 0 ? `${I18nManager.formatNumber(count)} ${I18n.t('users.users')}` : null}
```

Comment:
To Remove

Reply:
You want me to delete the carLicensePlate ?